CONTRIBUTING.md

```diff
@@ -1,4 +1,10 @@
-# 💥 How to Contribute
+# 💥 Prerequisite Skills to Contribute.
+- [Git](https://git-scm.com/)
+- [node.js](https://nodejs.org/en/)
```

Comment:
@shivam-sharma7 @vikasganiga05 using list still looks good but what do you think about using badges :thinking: 
https://shields.io/

Reply:
@vikasganiga05 i think it's looking good without badges.

Reply:
Yes, let it be list items it looks good :+1: